modules/developer_manual/pages/app/advanced/l10n.adoc

```diff
@@ -187,7 +187,7 @@ app.
 [[how-to-automatically-sync-translations]]
 == How to automatically sync translations
 
-1) Create a initial Transifex config within the app repository:
+1) Create a initial Transifex config within the app repository under "l10n/.tx/config":
```

Comment:
The sentence should start: "Create an initial".

Reply:
Shouldn't the path be an inline code block for proper highlighting instead of quotes?

Reply:
Either way's fine, but an inline code block would be that much clearer, yes.